docs/asyncapi.yaml

```diff
@@ -656,6 +656,10 @@ components:
       format: uuid
       description: GUID for a specific event reference
       example: 931e1a76-cff2-43ef-bfe5-82742eeeae68
+    RemainingAmount:
```

Comment:
async isn't rendering.

Plus you haven't referenced this schema to a message

Reply:
Appreciate we havva ticket to rectify the issues we have in our asyncs. However, I think we should be making an effort to add to them new items like this in the way that we want to going forward.

It reduces the overhead in the update later.

Plus, although these aren't great as we know, I still use them as the basis of my testing (ofc having to modify them).

My view is, you should defo add the RemainingAmount to the message it belongs (unless it doesn't exist, perhaps) and fix the async so it renders (providing it's a small fix, which I think it is)

Reply:
Unfortunately, the message body that RemainingAmount (Message -> DocumentSchedue) belongs does not exist as the moment, also there are other issues that needs resolving before it can render. Efforts to resolve all will take ticket beyond scope and allocated points, since there is already existing ticket to cover the issues, there is no point creating new ticket.